src/Microsoft.Restier.Core/ApiBase.cs

```diff
@@ -67,7 +67,14 @@ protected ApiConfiguration ApiConfiguration
                         var builder = this.ConfigureApi(new ApiBuilder());
```

Comment:
A side question here: I saw there is a class called `ContainerBuilder` in https://docs.asp.net/en/latest/fundamentals/dependency-injection.html?highlight=dependency%20injection#replacing-the-default-services-container. But I cannot find it in the latest source of DI. I wonder if the `IServiceCollection` and `ServiceCollection` now play the role of container here? Thanks.